Specifications/Services/ReverseCall/for_PingedConnection/when_pings_are_requested_every_5_seconds/given/all_dependencies.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.Runtime.Services.ReverseCalls.for_PingedConnection.given;
+using Dolittle.Runtime.Services.ReverseCalls.given;
+using Dolittle.Services.Contracts;
+using Google.Protobuf.WellKnownTypes;
+using Machine.Specifications;
+using mother_of_all_dependencies = Dolittle.Runtime.Services.ReverseCalls.given.all_dependencies;
+
+namespace Dolittle.Runtime.Services.ReverseCalls.for_PingedConnection.when_pings_are_requested_every_5_seconds.given
+{
+    public class all_dependencies : mother_of_all_dependencies
+    {
+        protected static Scenario scenario;
+        protected static a_message first_message_with_5_second_pings;
+        static object first_message_connect_arguments;
+        static ReverseCallArgumentsContext first_message_arguments_context;
+
+        Establish context = () =>
+        {
+            first_message_with_5_second_pings = new();
+            first_message_connect_arguments = new();
+            first_message_arguments_context = new()
```

Comment:
```suggestion

        Establish context = () =>
        {
            first_message_with_5_second_pings = new();
            var first_message_connect_arguments = new object();
            var first_message_arguments_context = new ReverseCallArgumentsContext()
```

Same thing here of converting unnecessary fields to vars.